openfl-tutorials/interactive_api/PyTorch_MedMNIST_2D/requirements.txt

```diff
@@ -0,0 +1 @@
+pip install medmnist
```

Comment:
If you want to pin specific versions of pillow or fire, you could specify them here. Also, the requirements file contains only package names, and then the entire file could be used as:  `pip install -r requirements.txt`

Reply:
Done, thanks.